packages/components/src/components/dialog/dialog.scss

```diff
@@ -73,7 +73,6 @@
             justify-content: space-between;
             align-items: center;
             margin-bottom: 1rem;
-            margin-top: 1.4rem;
```

Comment:
i removed this because this was making all the dialogs have an extra margin at the top, initially caused from an old PR: https://github.com/binary-com/deriv-app/pull/3719. Checked the reset password modal again after removing it and matches with design. Same with other error dialogs

<img width="586" alt="Screenshot 2023-08-22 at 3 06 19 PM" src="https://github.com/ameerul-deriv/deriv-app/assets/103412909/2b30a68e-733b-4d45-925b-49383e4c0903">
<img width="562" alt="Screenshot 2023-08-23 at 11 12 04 AM" src="https://github.com/ameerul-deriv/deriv-app/assets/103412909/d24a652c-9bf3-4b02-abfe-c49a2b30906a">
<img width="564" alt="Screenshot 2023-08-23 at 11 06 14 AM" src="https://github.com/ameerul-deriv/deriv-app/assets/103412909/3fe864d8-2285-45a7-b8bb-7774f7cc3a31">
<img width="978" alt="Screenshot 2023-08-23 at 11 28 52 AM" src="https://github.com/ameerul-deriv/deriv-app/assets/103412909/c9527e2f-fef1-4d2b-9a3d-747c5c865c28">

Reply:
design for reset password modal is here: https://www.figma.com/file/KEWV1ynSuyagoZIFX7nfoV/Release---v1.2.2---Sign-up%2FLog-in?type=design&node-id=744-19196&mode=design&t=wbXMly0Tp3dTp9Lh-0